operator/controllers/common/evaluationhandler.go

```diff
@@ -29,42 +29,21 @@ type EvaluationHandler struct {
 	SpanHandler ISpanHandler
 }
 
-type EvaluationCreateAttributes struct {
-	SpanName             string
-	EvaluationDefinition string
-	CheckType            apicommon.CheckType
-}
-
-func (r EvaluationHandler) ReconcileEvaluations(ctx context.Context, phaseCtx context.Context, reconcileObject client.Object, evaluationCreateAttributes EvaluationCreateAttributes) ([]klcv1alpha2.EvaluationStatus, apicommon.StatusSummary, error) {
+//nolint:gocognit,gocyclo
```

Comment:
with the refactoring do you still get errors for cyclomatic and cognitive complexity?